src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackPageUiTest.java

```diff
@@ -731,13 +736,18 @@ public void testDatePickerScripts() throws ParseException {
         
         ______TS("changing publish date does not affect visible date range publishTime > startTime");
         
+        cal.set(2014, 1, 19);
+        initialCal.setTime(cal.getTime());
+        feedbackPage.fillTimeValueForDatePickerTest(Const.ParamsNames.FEEDBACK_SESSION_STARTDATE, cal);
+        
+        
         cal.add(Calendar.DATE, 30);
         feedbackPage.fillTimeValueForDatePickerTest(Const.ParamsNames.FEEDBACK_SESSION_PUBLISHDATE, cal);
         
         //set the value back to start time
         cal.add(Calendar.DATE, -29);
```

Comment:
GitHub isn't showing it properly, what I mean is we no longer need to set the value back to start time and what you're doing is also more intuitive because adding 30 and then proceeding to revert it by 29 days seems prone to off by one error. Just using initial cal and checking using it seems like a better idea :)